File: split/geqdefine.pas
procedure geqdefine(p: halfword; t: quarterword; e: halfword);
begin
  eqdestroy(eqtb[p]);
  eqtb[p].hh.b1 := 1;
  eqtb[p].hh.b0 := t;
  eqtb[p].hh.rh := e;
end;

File: cpp/geqdefine.cpp
#include "geqdefine.h"
#include "eqdestroy.h"

void geqdefine(halfword p, quarterword t, halfword e)
{
	eqdestroy(eqtb[p]);
	subtype(p) = 1;
	type(p) = t;
	equiv(p) = e;
}
